Parser/pegen/parse.c

```diff
@@ -678,9 +678,9 @@ file_rule(Parser *p)
         void *a;
         Token * endmarker_var;
         if (
-            (a = statements_rule(p), 1)
+            (a = statements_rule(p), 1)  // statements?
```

Comment:
This looks kind of redundant.

Reply:
> This looks kind of redundant.

I thought about that, but it actually gives some extra translation: notice that the comment has the "?" at the end, which is the translation of the ,"1" in the rule that can be a bit obscure